addons/project/static/src/scss/project_rightpanel.scss

```diff
@@ -19,6 +18,10 @@ $o-rightpanel-filter-default-color: #D59244;
     }
 }
 
+html .o_web_client > .o_action_manager > .o_action > .o_content > .o_controller_with_rightpanel {
```

Comment:
Yes, otherwise the rule : https://github.com/odoo/odoo/blob/master/addons%2Fweb%2Fstatic%2Fsrc%2Fwebclient%2Fwebclient_layout.scss#L28
will have an higher priority.

Don't you agree ? :)

Reply:
Yes! Wow well, that's specific 🙂